<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>661</width>
      <height>411</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QGroupBox" name="groupBoxDevice">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>支持的音频输入设置</string>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>音频输入设备</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboDevices"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>221</width>
        <height>221</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>codec</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboCodec"/>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboSampleRate"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>SampleType</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboChannels"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Frequency (Hz)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Channels</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="comboSampleTypes"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Sample size(bits)</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Endianness</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="comboSampleSize"/>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="comboByteOrder"/>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>211</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>为了处理方便,本程序固定采样方式:

8000Hz,1 通道,audio/pcm

8 bits,UnSigned int,LittleEndian</string>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QGroupBox" name="groupBoxChart">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>300</height>
       </size>
      </property>
      <property name="title">
       <string>实时输入音频波形</string>
      </property>
      <widget class="QChartView" name="chartView">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>31</y>
         <width>800</width>
         <height>500</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>800</width>
         <height>500</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="LabBufferSize">
         <property name="text">
          <string>QAudioInput::bufferSize()=4000</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="LabBlockSize">
         <property name="text">
          <string>QIODevice数据块字节数=800</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>683</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actDeviceTest"/>
   <addaction name="actStart"/>
   <addaction name="actStop"/>
   <addaction name="separator"/>
   <addaction name="actQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actDeviceTest">
   <property name="text">
    <string>测试音频设置</string>
   </property>
   <property name="toolTip">
    <string>测试音频设置</string>
   </property>
  </action>
  <action name="actStart">
   <property name="text">
    <string>开始</string>
   </property>
   <property name="toolTip">
    <string>开始</string>
   </property>
  </action>
  <action name="actStop">
   <property name="text">
    <string>停止</string>
   </property>
   <property name="toolTip">
    <string>停止</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
